newsreader/src/main/java/life/qbic/newsreader/usermanagement/email/EmailSubmissionService.java

```diff
@@ -1,4 +1,4 @@
-package life.qbic.usermanagement;
+package life.qbic.newsreader.usermanagement.email;
```

Comment:
Why is the service for sending emails located in the news reader? I would expect it somewhere close to the domain but not the component that is supposed to consume the message and trigger an event. 
trigger an event != containing the service that is triggered